packages/dx-react-grid/src/plugins/search-panel.tsx

```diff
@@ -17,7 +18,10 @@ const defaultMessages = {
   searchPlaceholder: 'Search...',
 };
 
-export class SearchPanel extends React.PureComponent {
+class SearchPanelBase extends React.PureComponent<SearchPanelProps> {
+  static defaultProps: Partial<SearchPanelProps>;
```

Comment:
Can't field initialization be done here?
```js
static defaultProps: Partial<SeachPanelProps> = {
  messages: {},
};
```

Reply:
Yes, we can specify defaultProps in-place. I previously thought it will take a lot of space